Lib/test/test_typing.py

```diff
@@ -4250,7 +4254,6 @@ def test_generic_subclass_checks(self):
             with self.subTest(typ=typ):
                 self.assertRaises(TypeError, issubclass, typ, object)
                 self.assertRaises(TypeError, issubclass, typ, type)
-                self.assertRaises(TypeError, issubclass, typ, typ)
```

Comment:
Same here — the fact you've had to remove this assertion is quite concerning in terms of backwards compatibility :/

Reply:
I think my answer will be the same as for a similar comment above